packages/template-retail-react-app/app/components/store-locator/constants.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+export const STORE_LOCATOR_SUPPORTED_COUNTRIES = [
+    {
+        countryCode: 'US',
+        countryName: 'United States'
+    },
+    {
+        countryCode: 'DE',
+        countryName: 'Germany'
+    }
+]
+export const STORE_LOCATOR_DEFAULT_POSTAL_CODE = '10178'
+export const STORE_LOCATOR_RADIUS = 100
+export const STORE_LOCATOR_RADIUS_UNIT = 'km'
+export const STORE_LOCATOR_DEFAULT_COUNTRY = 'DE'
+export const STORE_LOCATOR_DEFAULT_COUNTRY_CODE = 'DE'
+export const STORE_LOCATOR_DEFAULT_PAGE_SIZE = 10
+export const STORE_LOCATOR_NUM_STORES_PER_REQUEST_API_MAX = 200 // This is an API limit and is therefore not configurable
```

Comment:
These values seems to be configs, and we expect customers to tweak them? If so, can we rename them to config.js?

Reply:
I agree these are more of configs rather than constants. But I'm trying to avoid introducing new patterns in this PR. We can discuss how we want to make it easier to configure features in a separate discussion.